amundsen_application/config.py

```diff
@@ -22,6 +31,28 @@ class Config:
 
     UNEDITABLE_SCHEMAS = set()  # type: Set[str]
 
+    # UNEDITABLE_TABLE_DESCRIPTION_MATCH_RULES is List of MatchRules
```

Comment:
I don't think we need these comments in the code and can clean this up. Setting `UNEDITABLE_TABLE_DESCRIPTION_MATCH_RULES = []  # type: List[MatchRuleObject]` give enough context about what this in.

Instead, this kind of information and examples would be written in the docs.